Xamarin.Forms.Core/FluentExtensions.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+
+namespace Xamarin.Forms.Fluent
+{
+	public static class FluentExtensions
+	{
+		#region BindableObject section
+
+		public static TView Fluent<TView>(this TView self,
+			Action<TView> action) where TView : BindableObject
+		{
+			if (self == null)
+				throw new ArgumentNullException(nameof(self));
+			if (action == null)
+				throw new ArgumentNullException(nameof(action));
+
+			action.Invoke(self);
+
+			return self;
+		}
+
+		public static TView Bind<TView>(this TView self,
+			BindableProperty targetProperty,
+			BindingBase bindingBase) where TView : BindableObject
+		{
+			if (self == null)
+				throw new ArgumentNullException(nameof(self));
+			if (targetProperty == null)
+				throw new ArgumentNullException(nameof(targetProperty));
+
+			self.SetBinding(targetProperty, bindingBase);
+
+			return self;
+		}
+
+		public static TView Bind<TView>(this TView self,
```

Comment:
this will get out of sync in no time. either provide the same subset of params as BindableObjectExtension.SetBinding does, or drop this.

Reply:
@StephaneDelcroix did you mean method below?

```csharp
public static TView Bind<TView>(this TView self,
			BindableProperty targetProperty,
			string path,
			BindingMode mode = BindingMode.Default,
			IValueConverter converter = null,
			object converterParameter = null,
			string stringFormat = null,
			object source = null) where TView : BindableObject
			=> Bind(self, targetProperty, new Binding
			{
				Path = path,
				Mode = mode,
				Converter = converter,
				ConverterParameter = converterParameter,
				StringFormat = stringFormat,
				Source = source
			});
```

Seems, it provides more options than BindableObjectExtension.SetBinding (addition **source** parameter)